tests/kafkatest/services/console_consumer.py

```diff
@@ -134,6 +141,14 @@ def prop_file(self, node):
         prop_file += str(self.security_config)
         return prop_file
 
+    def get_user_consumer_properties(self):
+        consumer_properties = ""
+
+        for key,value in self.consumer_properties.iteritems():
+            consumer_properties += " --consumer-property %s=%s" % (key, value)
```

Comment:
I'm not sure in this. So in case of two properties (k1=v1 and k2=v2) do you generate the `--consumer-property k1=v1 --consumer-property k2=v2` string? The repeated property didn't work when I tried it out and it only picked up the first one. 
I think you need to pass `--consumer-property "k1=v1,k2=v2"` as with some other commands.
In case the "k1=v1,k2=v2" format is needed, there is a nice one-liner way to do it:
```
','.join("%s=%r" % (key,val) for (key,val) in k.iteritems())
```

Reply:
I tested the kafka-console-producer that adding --producer-property key=value
```
bin/kafka-console-producer.sh  --broker-list test.com:9093 --topic test  --producer-property sasl.mechanism=TEST --producer-property security.protocol=TEST --producer-property sasl.kerberos.service.name=TEST
```
overrides the entries in the client properties as expected:
```
        sasl.kerberos.service.name = TEST
        ...
        sasl.mechanism = TEST
        security.protocol = TEST
```
I also verified that it works with kafka-console-consumer:
```
bin/kafka-console-consumer.sh  --bootstrap-server test:9093 --topic test  --consumer-property sasl.mechanism=TEST --consumer-property security.protocol=TEST --consumer-property sasl.kerberos.service.name=TEST --consumer.config /tmp/client.properties
```